config/config.yaml

```
@@ -108,6 +108,9 @@ storage:
     on_disk: false
     # Custom M param for hnsw graph built for payload index. If not set, default M will be used.
     payload_m: null
+  # Maximum number of concurrent updates to remote replica sets on other peers.
```

```suggestion
  # Maximum number of concurrent updates to replica sets.
```

It seems that a local replica set is also potentially targeted.

---

You're right. Sorry for not catching this in my initial suggestion to add this comment.